packages/shared/src/components/post/PostHeaderActions.tsx

```diff
@@ -136,9 +137,9 @@ export function PostHeaderActions({
         </SimpleTooltip>
       )}
       <PostOptionsMenu
-        onBookmark={onBookmark}
         onShare={onShare}
         post={post}
+        feedQueryKey={feedQueryKey}
```

Comment:
This could basically happen inside the component, as you already access the same context there.

Reply:
Moved it into the `PostOptionsMenu`